packages/ti_crowdstrike/manifest.yml

```
@@ -63,6 +63,7 @@ policy_templates:
             description: Token URL of CrowdStrike Falcon Intelligence.
             default: https://api.crowdstrike.com/oauth2/token
             required: true
+            secret: true
```

```suggestion
            secret: false
```
This isn't a secret.